shared/src/native-src/string.cpp

```diff
@@ -8,6 +8,18 @@
 
 namespace shared {
 
+    std::string PadLeft(const std::string& txt, std::size_t len, char c)
```

Comment:
can you add unit test here please ?https://github.com/DataDog/dd-trace-dotnet/blob/master/shared/test/Datadog.Trace.ClrProfiler.Native.Tests/string_test.cpp

Reply:
cannot you use something more like this ?https://stackoverflow.com/a/2366167

Reply:
That was my first option, but in Linux it didn't work (at least when I did it). I'll give it another try, though.

Reply:
Fixed in b20713cf19496c64313e59b3ad8874faad3cbb30